public/search.php

```diff
@@ -37,14 +39,36 @@
         $min_price = number_format($row['minimum_price']);
         $max_price = number_format($row['maximum_price']);
         $search_view = 'includes/search-valid.php';
+        $latitude = $row['latitude'];
+        $longitude = $row['longitude'];
     }
-
     db_query_error($sql_query);
 }
-
-
 ?>
 
-<?php require_once($search_view); ?>
+<div class="search-page-wrapper" id="wrapper">
+    <!-- sidemap -->
+    <div id="sidemap-wrapper">
+        <div class="sidemap-nav" id="sidemap-nav"></div>
+    </div>
+    <!-- /#sidemap-wrapper -->
+    <?php
```

Comment:
I would have preferred creating a custom map function in PHP so that we can reuse it for all the maps on the pages. However, since the map is will not be seen on the page unless the user clicks on `View Map`, it will be better to include this just before the footer.
```php
<?php require_once('includes/footer.php'); ?>
```